Service/Runner.php

```diff
@@ -104,14 +109,30 @@ public function __construct(
         $this->oauthClient = new Credentials($this->storageClient->getTokenString(), [
             'url' => $oauthApiUrl
         ]);
-        $this->oauthClient->enableReturnArrays(true);
+        $this->oauthClient3 = new Credentials($this->storageClient->getTokenString(), [
+            'url' => $this->getOauthUrl()
+        ]);
         $this->loggersService = $loggersService;
         $this->jobMapper = $jobMapper;
         $this->commandToGetHostIp = $commandToGetHostIp;
         $this->minLogPort = $minLogPort;
         $this->maxLogPort = $maxLogPort;
     }
 
+    /**
+     * @return string
+     */
+    private function getOauthUrl()
```

Comment:
tohle bych možná lehce přejmenoval, nebylo mi jasný, že to je pro ten novej oauth, kterej bude v services a furt jsem hledal, kde to pozná co je url na v2 a v3

Reply:
prejmenovany